bincache/Cargo.toml

```
@@ -6,17 +6,21 @@ description = "ZitaneLabs binary cache."
 
 [features]
 default = ["implicit-blocking"]
-implicit-blocking = []
-blocking = []
-rt_tokio_1 = ["dep:tokio"]
-rt_async-std_1 = ["dep:async-std"]
+implicit-blocking = ["dep:futures-io", "dep:futures-util", "async-compression/futures-io"]
+blocking = ["dep:futures-io", "dep:futures-util", "async-compression/futures-io"]
+rt_tokio_1 = ["dep:tokio", "async-compression/tokio"]
+rt_async-std_1 = ["dep:async-std", "dep:futures-io", "dep:futures-util", "async-compression/futures-io"]
```

I don't think we need the `futures` deps for `rt_async-std_1` anymore

```suggestion
rt_async-std_1 = ["dep:async-std", "async-compression/futures-io"]
```